### PR TITLE
Example of using the UsdSemantics schema

### PR DESCRIPTION
**Pull request by nvaszabo**
### Description of Change(s)
This PR adds a simple OpenUSD ASCII layer that exemplifies the features of the newly added UsdSemantics schema.
I have ensured that the sample file works in usdview

### Link to proposal ([if applicable](https://openusd.org/release/contributing_to_usd.html#step-1-get-consensus-for-major-changes))
https://openusd.org/dev/api/usd_semantics_overview.html

### Checklist

[x] I have created this PR based on the dev branch

[x] I have followed the [coding conventions](https://openusd.org/release/api/_page__coding__guidelines.html)

[ ] I have added unit tests that exercise this functionality (Reference: 
[testing guidelines](https://openusd.org/release/api/_page__testing__guidelines.html))

[ ] I have verified that all unit tests pass with the proposed changes

[x] I have submitted a signed Contributor License Agreement (Reference: 
[Contributor License Agreement instructions](https://openusd.org/release/contributing_to_usd.html#contributor-license-agreement))

### CODE REVIEW

File: extras/usd/examples/usdSemanticsExamples/bookshelf.usda [hunk 1]
```diff
@@ -0,0 +1,164 @@
+#usda 1.0
+(
+    doc = "This layer represents an example of using UsdSemantics"
+    upAxis = "Y"
+    startTimeCode = 0
+    endTimeCode = 200
+)
+
+def Xform "Booshelf" (
```
Comment (meshula) [inline extras/usd/examples/usdSemanticsExamples/bookshelf.usda]: A very halloween-y typo ;)

**Comment by jesschimein** [comment]
Filed as internal issue #USD-10386

**Comment by jesschimein** [comment]
Related to #3300

**Pull request merged**